lumigator/python/mzai/backend/backend/services/jobs.py

```diff
@@ -148,6 +148,10 @@ def _get_job_params(self, job_type: str, record, request: BaseModel) -> dict:
                 "top_p": request.top_p,
             }
 
+        # set max samples to unlimited if not specified
+        if request.max_samples is None:
+            request.max_samples = -1
+
```

Comment:
I think this code might be superfluous: it does not affect the returned `job_params` (as it is executed after setting it) and the reason why the test below succeeds is that we have also set a default value in the pydantic schema. Just to be sure, I have tried to run tests after removing this and they are still ok.

Reply:
fixed, thanks!